py3status/modules/wifi.py

```diff
@@ -184,17 +185,14 @@ def wifi(self):
                 signal_dbm = '? dBm'
                 signal_percent = '?%'
 
-            full_text = self.py3.safe_format(
-                self.format_up,
-                dict(
-                    bitrate=bitrate,
-                    signal_dbm=signal_dbm,
-                    signal_percent=signal_percent,
-                    ip=ip,
-                    device=self.device,
-                    icon=icon,
-                    ssid=ssid,
-                ))
+        full_text = self.py3.safe_format(self.format, dict(
+                                         bitrate=bitrate,
+                                         device=self.device,
+                                         icon=icon,
+                                         ip=ip,
+                                         signal_dbm=signal_dbm,
+                                         signal_percent=signal_percent,
+                                         ssid=ssid))
```

Comment:
imho the layout changes make this harder to read and update.

It is not immediately clear that bitrate etc are part of dict() rather than a parameter of safe_format().

being so indented to the right is just hard on the eyes.

`ssid=ssid))` this makes it hard to cleanly add/remove values.

Compare to deleted code above.